cli/command/container/create.go

```diff
@@ -243,7 +242,7 @@ func createContainer(ctx context.Context, dockerCli command.Cli, containerCfg *c
 			return err
 		}
 		if taggedRef, ok := namedRef.(reference.NamedTagged); ok && trustedRef != nil {
-			return trust.TagTrusted(ctx, dockerCli.Client(), dockerCli.Err(), trustedRef, taggedRef)
+			return image.TagTrusted(ctx, dockerCli, trustedRef, taggedRef)
 		}
```

Comment:
Looks like it passes with this revert; wondering if it's because the `.Client()` would be called as part of the `TagTrusted`, but now we're calling it early 🤔 (possibly before things related to trust were initialised?)

Reply:
☝️ So doesn't look like the cause, and very likely just a flaky test in general.